cbmc/proofs/IotMqtt_SubscribeSync/cbmc-batch.yaml

```diff
@@ -0,0 +1,8 @@
+build_memory: 32000
```

Comment:
Empty file with the one-line comment?  I think the Unsub proofs in PR do that now.

Reply:
Done!